cfme/services/catalogs/orchestration_template.py

```diff
@@ -18,7 +19,10 @@
 from cfme.utils.appliance.implementations.ui import navigator
 from cfme.utils.pretty import Pretty
 from cfme.utils.update import Updateable
+from cfme.utils.version import VersionPicker
 from widgetastic_manageiq import PaginationPane
+from widgetastic_manageiq import ReactCodemirror
```

Comment:
please use proper camel case naming

Reply:
@izapolsk  hm, this should be correct according to the name of the original component https://github.com/JedWatson/react-codemirror
and there it says `Codemirror Component for React.js`

Reply:
well, it's actually inconsistent there

Reply:
changed :)